cylc/flow/cfgspec/workflow.py

```diff
@@ -1339,7 +1339,17 @@ def get_script_common_text(this: str, example: Optional[str] = None):
 
             with Conf('remote', desc=DEPRECATION_WARN):
                 Conf('host', VDR.V_STRING)
-                Conf('owner', VDR.V_STRING)
+                # TODO: Convert URL to a stable or latest release doc after 8.0
+                # https://github.com/cylc/cylc-flow/issues/4663
+                Conf('owner', VDR.V_STRING, desc="""
+                    This setting is obsolete at Cylc 8.
+
+                    .. seealso::
+
+                        `Documentation on changes to remote owner
+                        <https://cylc.github.io/cylc-doc/nightly/html/
```

Comment:
```suggestion
                        <https://cylc.github.io/cylc-doc/latest/html/
```

Reply:
(explained in comment) won't work because the page isn't live yet

Reply:
@oliver-sanders I can't tell if you're justifying using `latest` or justifying using `nightly`

Reply:
latest won't work because the page is not published yet.

Reply:
But surely it doesn't matter that it doesn't work because we haven't released cylc-flow 8.0rc3 yet? Unless this for the benefit of internal MO testers?

Reply:
Ideally we don't want to leave it as nightly indefinitely because any changes we make to the docs on a daily basis could break doc links in the current cylc-flow release.

In fact, pinning to the release version would be better than using latest or nightly. If we could centralise all documentation links in a dict that could be achievable as part of the automated release process, maybe even linkchecking could be implemented

Reply:
> In fact, pinning to the release version would be better than using latest or nightly. If we could centralise all documentation links in a dict that could be achievable as part of the automated release process, maybe even linkchecking could be implemented

Yes, but a topic for another issue. I took a look at doing this a while back but it's trickier than it seems.

Reply:
I've written it up so we remember - https://github.com/cylc/cylc-flow/issues/4867

There is a nice solution, however, it involves work in both cylc-doc and cylc-flow which we don't have time for ATM.

Reply:
Since this didn't go into RC3 all this is now irrelevant!

Reply:
So, I guess the suggested change here should be committed before merging this PR?